pyanaconda/payload/dnf/payload.py

```diff
@@ -1908,14 +1908,16 @@ def _reload_treeinfo_repositories(self, base_repo_url=None):
         :param base_repo_url: base repository url. This is not saved anywhere when the function
                               is called. It will be add to the existing urls if not None.
         """
-        self._cleanup_old_treeinfo_repositories()
-        self._load_treeinfo_repositories(base_repo_url)
+        disabled_repo_list = self._cleanup_old_treeinfo_repositories()
+        self._load_treeinfo_repositories(base_repo_url, disabled_repo_list)
 
-    def _load_treeinfo_repositories(self, base_repo_url):
+    def _load_treeinfo_repositories(self, base_repo_url, disable_repositories):
```

Comment:
Maybe `disabled_repositories`? But I guess `disabled_names` would be less confusing.

Reply:
It's not disabled but a list you want to disable.

Reply:
Tried better name.